src/pretix/control/templates/pretixcontrol/pdf/index.html

```diff
@@ -415,6 +422,45 @@ <h2>{% trans "Welcome to the PDF ticket editor!" %}</h2>
                                 </div>
                             </div>
                         </div>
+                        <div class="col-sm-6">
+                            <div class="textcontainer">
+                                <div class="btn-group btn-group-justified" id="toolbox-verticalalign">
+                                    <div class="btn-group" role="group">
+                                        <button type="button" class="btn btn-default option toggling" data-action="top">
+                                            <span class="fa fa-sort-up"></span>
```

Comment:
To me it looks the other way around. The sort-up should make test align to bottom – to visually work though, the sort-up-arrow would need to be aligned at the bottom of the button. As an alternative, I tried `toggle-down`, which IMHO is (not exceptionally but a little) better to understand.

![Bildschirmfoto 2024-08-07 um 10 32 37](https://github.com/user-attachments/assets/00c4103c-eea9-4089-a356-569af7663bdc)

```suggestion
                                            <span class="fa fa-toggle-down"></span>
```